py-polars/tests/unit/operations/test_merge_sorted.py

```diff
@@ -172,11 +172,32 @@ def test_merge_sorted_parametric_string(lhs: pl.Series, rhs: pl.Series) -> None:
     assert_series_equal(merge_sorted, append_sorted)
 
 
+@given(
```

Comment:
Ideally, I would also like to test some more deeply nested structs. I don't think that will be any problems, but time has thought us that those are hard to get right.

Reply:
I updated `test_merge_sorted_parametric_struct` to have a nested struct.

`test_merge_sorted_self_parametric` also generates tests with nested structs, so I think we're covered.